scripts/patchup-example-names.mjs

```diff
@@ -1,17 +1,28 @@
 import fs from "fs/promises";
 import path from "path";
 
-let dirPath = path.join(process.cwd(), "examples");
-let dir = await fs.readdir(dirPath);
+let examplePath = path.join(process.cwd(), "examples");
+let templatePath = path.join(process.cwd(), "templates");
+let examples = await fs.readdir(examplePath);
+let templates = await fs.readdir(templatePath);
 
-dir.forEach(async (name) => {
-  let fullPath = path.join(dirPath, name);
+async function update(dir, name) {
```

Comment:
Let's make `base` an arg to this function and pass it explicitly instead of deriving it from the parent dir name.